images/2018-04-10-personalized-search/PersonalizationComponent.java

```diff
@@ -0,0 +1,205 @@
+package no.finntech.search.solr.personalization;
```

Comment:
Hm.  er egentlig for, men er det greit å gi ut denne i sin helhet?

Fint om du/vi tar emn diskusjon.

momenter:  ansvar ved feil, lisens, kredittering, intelectual property

Reply:
hmm, tenkte umiddelbart det var greit, men har jeg ikke tenkt på noen av de momentene. skal se hva andre her på huset har gjort.

Reply:
fjerner det nå, så sjekker jeg ut om dette er ok med folk i morgen, og legger det isåfall på da.
så kan jeg legge ut blogginnlegget nå :)